api/sms_provider/sms_provider.go

```diff
@@ -36,6 +36,8 @@ func GetSmsProvider(config conf.Configuration) (SmsProvider, error) {
 		return NewTextlocalProvider(config.Sms.Textlocal)
 	case "vonage":
 		return NewVonageProvider(config.Sms.Vonage)
+	case "custom":
+		return NewCustomProvider(config.Sms.Custom)
```

Comment:
Custom is a fairly generic name, could you maybe use something that describes it more closely. `request` may be a good name?

Reply:
Auth0 uses `sms_gateway` which would duplicate the `SMS` prefix. We could switch to `gateway`.